src/cfn-registry.ts

```diff
@@ -0,0 +1,36 @@
+import * as AWS from 'aws-sdk';
+import { createAwsClient } from './aws';
+
+export async function getResourceDefinitonFromRegistry(name: string, _version?: string): Promise<AWS.CloudFormation.DescribeTypeOutput> {
```

Comment:
Since this is already in the context of `cfn-registry.ts`, the function name can be more concise:

```suggestion
export async function describeResourceType(typeName: string, _version?: string): Promise<AWS.CloudFormation.DescribeTypeOutput> {
```